sdk/servicebus/Microsoft.Azure.ServiceBus/src/SessionHandlerOptions.cs

```diff
@@ -53,7 +53,7 @@ public TimeSpan MaxAutoRenewDuration
             }
         }
 
-        /// <summary>Gets or sets the time to wait for receiving a message.</summary>
+        /// <summary>Gets or sets the time to wait for receiving a message. This is the timeout the session is going to wait for the next available message before closing. This value has an impact on the message throughput.</summary>
```

Comment:
>This value has an impact on the message throughput [](start = 172, length = 50)

Could we add more details here as well? i.e., what does the customer have to do to fix the throughput? How does this impact? What would it mean to increase or decrease it?